assets/blueprints/account/src/lib.rs

```diff
@@ -13,10 +13,10 @@ blueprint! {
             .instantiate();
 
             let access_rules = AccessRules::new()
-                .method("balance", rule!(allow_all))
-                .method("deposit", rule!(allow_all))
-                .method("deposit_batch", rule!(allow_all))
-                .default(withdraw_rule);
+                .method("balance", rule!(allow_all), LOCKED)
+                .method("deposit", rule!(allow_all), LOCKED)
+                .method("deposit_batch", rule!(allow_all), LOCKED)
+                .default(withdraw_rule, LOCKED);
```

Comment:
Not sure if we want the ability to mutate default rule, what was the idea behind this?

Reply:
Josh and I discussed this during a call. We agreed that default case on access rules should not be mutable and that only methods and groups should be mutable. This has been addressed in commit [6d81f6a](https://github.com/radixdlt/radixdlt-scrypto/pull/624/commits/6d81f6aa6383f4443c18c87fa04b2a9bbe55cc7f).